tracpro/polls/charts.py

```diff
@@ -103,22 +107,52 @@ def multiple_pollruns_multiple_choice(pollruns, answers):
     }
 
 
-def multiple_pollruns_numeric(pollruns, responses, answers, question):
+def multiple_pollruns_numeric(pollruns, responses, answers, question, split_regions):
     """Chart data for multiple pollruns of a poll."""
-    answer_sums, answer_avgs = answers.get_answer_summaries()
-    response_rates = responses.get_response_rates()
+    if split_regions:
```

Comment:
This function acts in two very different ways based on the value of `split_regions`. I think it would be an easier read if split into two functions.